hapi/src/routes/create-faucet-account.route.js

```diff
@@ -43,25 +44,35 @@ module.exports = {
         eosConfig.faucet.password
       )
 
+      await sleepFor(1)
+
       const {
-        data: { account_name: account, permissions }
+        data: { accounts }
       } = await axiosUtil.instance.post(
-        `${eosConfig.apiEndpoint}/v1/chain/get_account`,
+        `${eosConfig.apiEndpoint}/v1/chain/get_accounts_by_authorizers`,
```

Comment:
Why is this required?

Reply:
this endpoint is not related to `newnonebact`, it is in the api node section which is different from the system contracts.